src/store/adjustVault.ts

```diff
@@ -7,7 +7,7 @@ import {
 import {
   floorMultiplyBy,
   ratioGTE,
-} from '@agoric/zoe/src/contractSupport/ratio';
+} from '@agoric/zoe/src/contractSupport/index';
```

Comment:
this is in ERTP now. you might not need `@agoric/zoe` anymore

```suggestion
import { floorMultiplyBy, ratioGTE } from '@agoric/ertp/src/ratio.js';
```

Reply:
Reverted to older version of ertp so this won't be included